Makefile

```diff
@@ -33,7 +33,7 @@ run: build
 	@./$(TARGET)
 
 docker: clean lint
-	@docker build -f build/dev.Dockerfile . -t persona-id/proxysql-agent:latest
+	@docker build -f build/dev.Dockerfile -t persona-id/proxysql-agent:latest -t persona-id/proxysql-agent:1.1.0 .
```

Comment:
Do you still want this?

Reply:
nope that was for testing internally, good catch